Euler31.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[58]:
+
+
+target_balance = 200                              # Calculating for 200
+denominations = [1, 2, 5, 10, 20, 50, 100, 200]   # Donimanations: [1, 2, 5, 10, 20, 50, 100, 200]
```

Comment:
Also, you don't have to write comments on every line. As you can see, "# Donimanations: [1, 2, 5, 10, 20, 50, 100, 200]" and "# Calculating for 200" are unnecessary.